examples/spread-in-binding-patterns/spread_in_binding_patterns.md

```diff
@@ -0,0 +1,7 @@
+# Rest binding pattern
+
+You can use the rest binding pattern (...r) to bind the fields that are not explicitly bound in the binding pattern. This can be used in mapping, list, and error binding patterns.
```

Comment:
```suggestion
You can use the rest binding pattern (`...r`) to bind the fields that are not explicitly bound in the binding pattern. This can be used in mapping, list, and error binding patterns.
```